### ISSUE TITLE
Do not create Wombat error when no Konnektive orders are found

### ISSUE THREAD

**Issue by onedanshow** [Woodbolt-International/konnektive_integration#8]
Please set it up to not create a Wombat error when no orders are found in Konnektive

![screen shot 2015-05-30 at 12 47 58 pm](https://cloud.githubusercontent.com/assets/149438/7898586/2b210a4c-06ca-11e5-9e25-3ce182310b14.png)

**Comment by apeacox** [comment]
This problem should be fixed now. Unfortunately I had to write an ugly hack to fix this. The problem is due to Konnektive API response. In case it doesn't found any data for a given date, it replies with:

```
{
result: "ERROR",
message: "No orders matching those parameters could be found"
}
```

I decided to handle all error messages from konnektive api by replying with HTTP 500 and the message returned from the api, in this way wombat would have tracked any error from the integration.

In the end, to fix this specific case, I've added an `if` statement to eventually check if the error message *is equal to* `No orders matching those parameters could be found`  and in that case I will return an empty array. If the error message is different, it will continue to respond with HTTP 500 along with the message, so that Wombat can still track any other errors from integration.
